rpm/Makefile

```diff
@@ -1,11 +1,12 @@
+ALPINE:=$(shell $(CURDIR)/../detect_alpine_image)
```

Comment:
For reference this relates to https://github.com/docker/docker-ce-packaging/pull/10